.vscode/cspell.json

```diff
@@ -45,6 +45,7 @@
     "dotenv",
     "downcasted",
     "downcasting",
+    "dtolnay",
```

Comment:
This is no longer necessary. Once removed, run spell check again and fix any additional errors.